### PR TITLE
Fixes some closets not taking things from the turf

### LINKED ISSUE #25720: Hyperfractal Gigashuttle Lockers require you right click to pickup gear from lockers. Can't just normally left click
I dunno... I just keep reporting bugs the freeze never ends

### PR DESCRIPTION
**Pull request by Cyberboss**
I think I need to rethink late initialization...

Fixes #25720

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -37,11 +37,7 @@
 
 /obj/structure/closet/Initialize(mapload)
 	if(mapload && !opened)		// if closed, any item at the crate's loc is put in the contents
-		. = TRUE	//late initialize so we come back here
-	else if(initialized)
-		//we were late initialized, so now it's time to...
-		take_contents()
-		return
+		addtimer(CALLBACK(src, .proc/take_contents), 0)
```
Comment (coiax) [inline code/game/objects/structures/crates_lockers/closets.dm]: shouldn't this be INVOKE_ASYNC?

Comment (Cyberboss) [inline code/game/objects/structures/crates_lockers/closets.dm]: No, because we want it to wait for everything

**Comment by Iatots** [comment]
I don't think its just the closets, most containers seem to be broken at the moment if they come from a shuttle, for instance the scrapheap: there are two medkits, but their content can't be interacted with, not even once dumped on the floor.

**Comment by Cyberboss** [comment]
All containers are closets

**Pull request merged**